docs/content/configuration/storage.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Configuring Storage
+description: Configuring Storage in Athens
+---
+
+## Storage
+
+In project we support many storage types:
+
+1. [Memory](#memory) - `memory`
```

Comment:
![image](https://user-images.githubusercontent.com/1522652/48247091-52783300-e3f2-11e8-8c97-0fee52c1ae2c.png)
This looks weird to me, can you get rid of ticked names and a comment, you mention it in DO Section either way

Reply:
you can mention it in a section but not in a config, it might lead to a misunderstanding that storage type declaration is part of the configuration file
example: 

```
## Memory
### Storage type: memory
This storage don't need an
```

Reply:
But `StorageType` is part of configuration file and user should set it if he wanna use proper driver

Reply:
oh right, it is unparented. 👍 for keeping it there then, sorry for misunderstanding

Reply:
So we shouldn't change there anything? This StorageTypes in configuration are already in this PR

Reply:
yes, please keep it as is 
after addressing @manugupt1 's comment i'm happy with this PR